docs/reference/koin-core/modules.md

```diff
@@ -253,20 +253,20 @@ startKoin { modules(featureModule1, featureModule2) }
 Notice that all modules will be included only once: `dataModule`, `domainModule`, `featureModule1`, `featureModule2`.
 
 :::info
-    If you have any compiling issue while including modules from the same file, either use `get()` Kotlin attribute operator on your module eitehr separate each moduel in files. See https://github.com/InsertKoinIO/koin/issues/1341 workaround
+    If you have any compiling issue while including modules from the same file, either use `get()` Kotlin attribute operator on your module either separate each module in files. See https://github.com/InsertKoinIO/koin/issues/1341 workaround
```

Comment:
This sentence reads kinda screwy to me. I think this makes sense and is what the author intended?
```suggestion
    If you have any compiling issue while including modules from the same file, use the `get()` Kotlin attribute operator on each module. See https://github.com/InsertKoinIO/koin/issues/1341 workaround
```

Reply:
I believe that this note pertains to the creation of modules within the same Kotlin file. This practice can sometimes cause issues in compilation time, and the note intends to inform the developer about it. It recommends separating the modules into distinct files if this kind of problem is happening. At least it was what I understood.

Was this indeed your intention, @arnaudgiuliani?

Reply:
yes, lets keep to say we have both possibility to use get() or separate file. It's more a IDE/Kotlin compilation stuff problem with our `include` operator